src/Authorization/Services/Implementation/ResourceRegistryWrapper.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.IO;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Altinn.Authorization.ABAC.Xacml;
+using Altinn.Platform.Authorization.Clients;
+using Altinn.Platform.Authorization.Helpers;
+using Altinn.Platform.Authorization.Services.Interface;
+
+namespace Altinn.Platform.Authorization.Services.Implementation
+{
+    /// <summary>
+    /// Wrapper for resource registry
+    /// </summary>
+    public class ResourceRegistryWrapper : IResourceRegistry
+    {
+        private readonly ResourceRegistryClient _client;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ResourceRegistryWrapper"/> class.
+        /// </summary>
+        /// <param name="resourceRegistryClient">The httpclient</param>
+        public ResourceRegistryWrapper(ResourceRegistryClient  resourceRegistryClient)
+        {
+            _client = resourceRegistryClient;
+        }
+
+        /// <inheritdoc/>
+        public async Task<XacmlPolicy> GetResourcePolicyAsync(string resourceId)
+        {
+            XacmlPolicy policy = null;
+            string apiurl = $"resorces/{resourceId}/policy";
```

Comment:
Typo in apiurl: resorces
Also this endpoint don't exist yet in the resourceregistry I think?

Reply:
Fixed typo. Endpoint do exist now  https://github.com/Altinn/altinn-resource-registry/blob/main/src/ResourceRegistry/Controllers/ResourceController.cs#L126